Consul/Operator.cs

```diff
@@ -249,6 +280,25 @@ public Task<QueryResult<string[]>> SegmentList(CancellationToken ct = default)
         {
             return SegmentList(QueryOptions.Default, ct);
         }
+
+        /// <summary>
+        /// CreateArea will create a new network area. The ID in the given structure must
```

Comment:
here as well -> ` The ID in the given structure must be empty and a generated ID will be returned on success.`

Reply:
Done, thanks.